scripts/list_prompts.py

```
@@ -0,0 +1,142 @@
+            fields.update(self._extract_jinja_vars(content['input_format']))
+            
+        # Check for explicit input_fields declaration
+        if 'input_fields' in content:
+            fields.update(content['input_fields'])
+            
+        return sorted(fields)
+        
+    def _extract_jinja_vars(self, text: str) -> set:
+        """Extract Jinja2 variable names from text."""
+        pattern = r'\{\{\s*([a-zA-Z_][a-zA-Z0-9_]*)\s*\}\}'
+        return set(re.findall(pattern, text))
```

Ruff (0.11.9)

Undefined name `re`

(F821)